docs/conversation.md

```diff
@@ -330,6 +331,36 @@
         + `401 Unauthorized` When the participant is a guest
         + `404 Not Found` When the conversation could not be found for the participant
 
+## Set time to live of messages for a conversation
+
+* Required capability: `time-to-live`
+* Method: `POST`
+* Endpoint: `/room/{token}/ttl`
+* Data:
+
+    field | type | Description
+    ---|---|---
+    `ttl` | int | The time to live of messages in seconds. Must be greater than zero.
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `400 Bad Request` Invalid value
+        + `403 Forbidden` When the current user is not a moderator, owner or guest moderator
+        + `404 Not Found` When the conversation could not be found for the participant
+
+## Disable the time to live of messages for a conversation
+
+* Required capability: `time-to-live`
+* Method: `DELETE`
+* Endpoint: `/room/{token}/ttl`
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `403 Forbidden` When the current user is not a moderator, owner or guest moderator
+        + `404 Not Found` When the conversation could not be found for the participant
```

Comment:
Since 0 means disabled, we can also just do a post with 0? Less endpoints less breaking?